docs/src/pages/index.tsx

```diff
@@ -14,6 +31,7 @@ import { Homepage } from "@site/src/components/home";
 export default function Home(): React.ReactElement {
 	return (
 		<Layout>
+			<Suspense fallback={<div />}></Suspense>
```

Comment:
Is there any loading element we can use here other than just an empty div?

Reply:
Since we only expect this to appear during local development, leaving an explicit notice with more details sounds like a good idea to me.